### DISCUSSION TITLE
Comments on a file in a PR should remain inline with the file if the file is renamed

### DISCUSSION THREAD

**Discussion by tylerhains** [community/community#52757] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Git is smart enough to figure out when I'm renaming a file as opposed to deleting one file and creating another one exactly like it. As such, if we have comments on the file before it's renamed, they really should remain inline on the file in the PR after the file is renamed.

**Comment by osortega** [comment]
Hello @tylerhains!

This might be something more generally related to Pull Requests rather than Codespaces.
Do you mind please changing the category if that is the case?

Thanks!
